### ISSUE TITLE
Force CRC value

### ISSUE THREAD

**Issue by IceLancerSR** [mluzarow/fake-game-generator-hyperspin#1]
This thing would be very welcomed, would u know how to write it to this py?
So it will read crc values from xml's and write them to "fake files" ?

**Comment by mluzarow** [comment]
Yeah; I suppose that wouldn't be too difficult. How would you you like the output files to be structured? Something like "crc_value game_name.txt"?

**Comment by IceLancerSR** [comment]
When it writes txt files with names to apply crc value to those files the match from xml.It doesnt have to be txt , a zip would do fine as well. As long as crc is applied to them so it matched the xml data

**Comment by mluzarow** [comment]
How's this: #2 ?

**Comment by IceLancerSR** [comment]
Well not quite what i had in mind. But damn u are fast :)
Right now 01A4B1E9 Yoshi's Story (USA) (En,Ja)
But the crc value is suppose to be forced into the file
Like this:
https://www.nayuki.io/page/forcing-a-files-crc-to-any-value

So when u write Yoshi's Story (USA) (En,Ja).txt and when i look its crc value in checksum it needs to be what xml says.So it will mimic original file exactly including the crc file

**Comment by mluzarow** [comment]
Oh I see what you mean now; sorry about the confusion.

**Comment by IceLancerSR** [comment]
All good :) This does seem like heavy job to do.If you could pull it off it could save people a lot of hustle. Simply because scrapers right now do not match file name ,they search by crc value only and then rip the data for it. And with a lot of custom homebrew games database stays empty, heaving something like this would be immense help.

**Comment by mluzarow** [comment]
Alrighty; has this #2 ? Sorry for the delay; work and all. :>

**Comment by IceLancerSR** [comment]
Damn this is amazing work. I am merely a hack when it comes to py. So i really congratulate u on ur knowledge! This is doing perfect job! I appreciate taking finding time !
By looking at the code, this is level way beyond me..maybe one day.. 

P.S
i was wondering what is your knowledge with VB? Microsoft VisualBasic

**Comment by mluzarow** [comment]
I've built VB stuff in the past but that was a long time ago. I'll try to look into it and see what I can do, though; sounds like it might be a simple edit.

**Comment by IceLancerSR** [comment]
Man if u do manage to do that i will find a way buy u a beer or w/e drink of ur choice! 
Not to mention how simpler things will be with data management without loosing all those good synopses aka game detail info

**Comment by mluzarow** [comment]
Can you give me an example of a ROMs XML that uses those new columns? All of the ones on Hyperlist (which I assume is the source) do not have them.

**Comment by IceLancerSR** [comment]
Sorry for late reply .. here is format 
`	<game image="1" index="true" name="1/Ace Attorney Investigations - Miles Edgeworth (USA)">
		<description>Ace Attorney Investigations - Miles Edgeworth (USA)</description>
		<synopsis>Ace Attorney Investigations: Miles Edgeworth is the fifth game in the Ace Attorney series. It is an interquel set between Phoenix Wright: Ace Attorney Trials and Tribulations and Apollo Justice: Ace Attorney.
This game is a standard "point and click" adventure, with the player controlling Edgeworth's movements and what he interacts with. For each case, the goal is to collect evidence to determine who the perpetrator is, how the crime was committed and their motive. This is presented as a combination of a third-person view showing the characters at the crime scene and through animated close-up images of the characters, annotated by text dialog. In some cases, still images are used to highlight dramatic scenes.</synopsis>
		<score>0.7</score>
		<year>16/02/2010</year>
		<developer>Capcom</developer>
		<manufacturer>Capcom</manufacturer>
		<genre>Adventure</genre>
		<players>1</players>
		<enabled>Yes</enabled>
	</game>`
or better yet here is one of file examples
its basically the same as the ones u have in this git except for 3 new columns
[Sega Mega Drive.zip](https://github.com/mluzarow/fake-game-generator-hyperspin/files/1610136/Sega.Mega.Drive.zip)

**Comment by IceLancerSR** [comment]
WOW i must be blind, i was watching like a hawk and i didn't see when u made a fork and changes.. Is it done? :) Or should i try and grab in attempt to compile?

**Comment by mluzarow** [comment]
Yeah; I never actually tested it, haha.

**Comment by IceLancerSR** [comment]
I ran into other problems once i added new cells, it needs bunch more stuff to make new cells, cant just add them..

**Comment by IceLancerSR** [comment]
AH! Eureka!
I made new columns but i was idiot enough not to check if those column numbers already exist.. so it was clashing.
Its loading synapses now and its even showing.But i made mess somewhere ,because its loading synapses in developer column and developer in synaps

**Comment by IceLancerSR** [comment]
Stupid copy /paste! Its working 
bunch of wrong copy paste , same name == bad result , fixed it , all is good ! AMAZING!

**Comment by IceLancerSR** [comment]
I promised a drink of ur choice.. i just have to find the way to get it to you ! :) i feel i kinda bonded with ur persona , if not all good anyway , feel free to contact me, or not all good! Thanks !

**Comment by IceLancerSR** [comment]
P.S an amazing idea would be to add one more option to this.. Export in different format.. for emulationstation example. Now that is a challenge ..

**Comment by mluzarow** [comment]
Haha; I barely did anything. It was all you!

**Comment by IceLancerSR** [comment]
Hey there PY master, i need your expertise..  i wrote some scripts and one is causing me serious headache because its way to slow.. it took 2 days to do one big XML
If you could possibly take a look?
I added some info in rar
[Small request.zip](https://github.com/mluzarow/fake-game-generator-hyperspin/files/1617409/Small.request.zip)
